services/brig/src/Brig/Team/Email.hs

```diff
@@ -56,6 +54,13 @@ sendMemberWelcomeMail to tid teamName loc = do
   let mail = MemberWelcomeEmail to tid teamName
   liftSem $ sendMail $ renderMemberWelcomeMail mail tpl branding
 
+sendInvitationMailPersonalUser :: (Member EmailSending r) => EmailAddress -> TeamId -> EmailAddress -> InvitationCode -> Maybe Locale -> (AppT r) ()
```

Comment:
how many more instances of this pattern before it's time to write a generic `sendSomeMail` function?  :)